### PR TITLE
Ansible Tower job_wait module

### PR DESCRIPTION
**Pull request by wwitzel3**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

Ansible Tower job wait module. (tower_job_wait)
##### ANSIBLE VERSION

```
ansible 2.3.0 (tower_module_job_wait 867069163a) last updated 2017/03/01 15:12:50 (GMT -400)
  config file =
  configured module search path = Default w/o overrides
```

##### SUMMARY

Allows playbook authors to wait for and react to job status and run details.

### CODE REVIEW

File: lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_wait.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      description:
+        - ID of the job to monitor.
+      required: True
+    min_interval:
+      description:
+        - Minimum interval to request an update from Tower.
+      default: 1
+    max_interval:
+      description:
+        - Maximum interval to request an update from Tower.
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_wait.py]: in seconds?

File: lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_wait.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    - Wait for Ansible Tower job to finish and report success or failure. See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    job_id:
+      description:
+        - ID of the job to monitor.
+      required: True
+    min_interval:
+      description:
+        - Minimum interval to request an update from Tower.
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_wait.py]: in seconds?

File: lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_wait.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    )
+
+    if not HAS_TOWER_CLI:
+        module.fail_json(msg='ansible-tower-cli required for this module')
+
+    json_output = {}
+    fail_json = None
+
+    # tower-cli gets very noisy when monitoring.
+    # We use nostdout to surppress the stdout during our monitor call.
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_wait.py]: supress

**Comment by matburt** [comment]
shipit

**Comment by ghjm** [comment]
shipit

**Comment by jlaska** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Pull request merged**